plugins/modules/ansible_galaxy_install.py

```diff
@@ -27,6 +27,11 @@
 requirements:
   - Ansible 2.9, ansible-base 2.10, or ansible-core 2.11 or newer
 options:
+  token:
+    description:
+      - The Ansible Galaxy API key which can be found at https://galaxy.ansible.com/me/preferences.
+      - Can be used to install collections and roles from private git repositories.
```

Comment:
I'm not sure how the Ansible Galaxy token is related to private git repositories. The token can be useful when talking to private galaxy_ng / Automation Hub instances, AFAIK, but I don't see how it is related to git.

(Also the Ansible Galaxy token from https://galaxy.ansible.com/me/preferences should only ever be needed to publish collections or roles, but not to install them. Or do I miss something?)